lib/src/layout/sidebar/sidebar_items.dart

```diff
@@ -290,7 +292,13 @@ class _SidebarItemState extends State<_SidebarItem> {
         break;
     }
 
-    Widget result = Semantics(
+    Color backgroundColor() {
+      if (_isDraggingInside) return MacosColors.controlAccentColor;
```

Comment:
Let's use `MacosTheme.of(context).primaryColor` for the drop background and line colors.